### PR TITLE
WIP: Bump to 2.12 + add aarch64

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Seems like we'll need to wait until the release of tensorflow 2.12 (as we only have tensorboard 2.12 for aarch64). This is scheduled to happen at the end of this month: https://discuss.tensorflow.org/t/when-will-tensorflow-2-12-be-released/14717

**Comment by Tobias-Fischer** [comment]
Yes please, go for it!

**Comment by Tobias-Fischer** [comment]
Note that we'll need to update keras as well ..

**Comment by Tobias-Fischer** [comment]
Good point actually - afaik keras sits on top of tendorflow, so we might want to remove the keras host dep (at least to play around on this PR)?

File: conda-forge.yml [hunk 1]
```diff
@@ -3,12 +3,9 @@ build_platform:
 conda_forge_output_validation: true
 provider:
   win: azure
+  linux_aarch64: azure
 github:
   branch_name: main
   tooling_branch_name: main
 conda_build:
   pkg_format: '2'
-# https://github.com/conda-forge/conda-smithy/pull/1686
```
Comment (hmaarrfk) [inline conda-forge.yml:11]: I don't think a new version of conda-smithy has been released. can we keep this please.

Could you also:

```
skip: true  # [no aarch64]
```

and maybe everything except python 3.9 for the next build.

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin please rerender

**Comment by Tobias-Fischer** [comment]
I might be wrong, but isn't the issue further up?

```
WARNING: Download from https://storage.googleapis.com/mirror.tensorflow.org/github.com/tensorflow/runtime/archive/91d765cad5599f9710973d3e34d4dc22583e2e79.tar.gz failed: class java.io.FileNotFoundException GET returned 404 Not Found
...
WARNING: Download from https://storage.googleapis.com/mirror.tensorflow.org/github.com/llvm/llvm-project/archive/10939d1d580b9d3c9c2f3539c6bdb39f408179c0.tar.gz failed: class java.io.FileNotFoundException GET returned 404 Not Found
...
WARNING: Download from https://storage.googleapis.com/mirror.tensorflow.org/github.com/google/XNNPACK/archive/659147817805d17c7be2d60bd7bbca7e780f9c82.zip failed: class java.io.FileNotFoundException GET returned 404 Not Found
```

Seems like we have to wait for a RC after all.

**Comment by Tobias-Fischer** [comment]
Let's try https://github.com/search?q=repo%3Atensorflow%2Ftensorflow%20elinux_aarch64&type=code

**Comment by Tobias-Fischer** [comment]
This weird protobuf error now shows up everywhere - this commit introduced it: https://github.com/tensorflow/tensorflow/commit/84f40925e929d05e72ab9234e53c729224e3af38

**Comment by Tobias-Fischer** [comment]
I don't understand bazel, but this looks like it could be helpful: https://github.com/envoyproxy/envoy/pull/23345

**Comment by Tobias-Fischer** [comment]
Feel free to play around here @ngam :)

**Comment by Tobias-Fischer** [comment]
Same problem still in rc1:
```
# Configuration: 3f443e42aa2f8201efa7964106f941d560d07bafca668c8eca9133ef11e55c03
# Execution platform: @local_execution_config_platform//:platform
In file included from bazel-out/k8-opt/bin/tensorflow/core/framework/step_stats.pb.cc:4:
bazel-out/k8-opt/bin/tensorflow/core/framework/step_stats.pb.h:10:10: fatal error: google/protobuf/port_def.inc: No such file or directory
   10 | #include <google/protobuf/port_def.inc>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by hmaarrfk** [comment]
Any objections to moving forward with https://github.com/conda-forge/tensorflow-feedstock/pull/303 while some issues with 2.12 are resolved?

**Comment by Tobias-Fischer** [comment]
Go for it :)

**Pull request closed without merging**